src/mapboxgl.css

```diff
@@ -1,3 +1,5 @@
+/* TODO */
```

Comment:
This might be worth moving to a separate / new issue, describing the TODO in more detail.